libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -4745,6 +4746,44 @@ void GCS_MAVLINK::send_generator_status() const
 #endif
 }
 
+void GCS_MAVLINK::send_water_depth() const
```

Comment:
should be ifdef for vehicle rover

Reply:
added #if APM_BUILD_TYPE(APM_BUILD_Rover) so that "send_water_depth()" only does something for Rover (for other vehicles the method is empty).